save-backend/src/main/kotlin/com/saveourtool/save/backend/service/UserDetailsService.kt

```diff
@@ -22,22 +26,30 @@ import java.util.*
 @Service
 class UserDetailsService(
     private val userRepository: UserRepository,
+    private val originalLoginRepository: OriginalLoginRepository,
 ) : ReactiveUserDetailsService {
     private val logger = LoggerFactory.getLogger(javaClass)
 
     override fun findByUsername(username: String): Mono<UserDetails> =
-            { userRepository.findByName(username) }
-                .toMono()
-                .getIdentitySourceAwareUserDetails(username)
+            userRepository.findByName(username)?.let {
```

Comment:
1) All the logic should be unified:
```Mono.just()
          .map {
                
          }
```
2) You duplicate same call twice: ` .getIdentitySourceAwareUserDetails(username)`